packages/zwave-js/src/lib/controller/Controller.ts

```diff
@@ -1283,6 +1293,46 @@ export class ZWaveController
 		this._nodeType = initData.nodeType;
 		this._supportsTimers = initData.supportsTimers;
 		// ignore the initVersion, no clue what to do with it
+
+		// BUGBUG: Dummy check? SimplicityStudio adds this extra 3 argument to this command. I suspect that's not the secret sauce for LR inclusion, but S2 encryption is.
+		await this.driver.sendMessage<
+			GetSerialApiCapabilitiesResponse
+		>(
+			new GetSerialApiCapabilitiesRequest(this.driver, {mysteryValue: 3}),
+			{
+				supportCheck: false,
+			},
+		);
+
+		// fetch the list of long range nodes until the controller reports no more
+		const lrNodeIds: Array<number> = [];
+		let lrChannel : LongRangeChannel | undefined;
+		const maxPayloadSize = await this.getMaxPayloadSize();
+		let maxPayloadSizeLR : number | undefined;
```

Comment:
Do we want to cache these off into member variables, or just report them for now?

I did NOT include APIs for setting some of these things, as I don't know what we intend to expose in JS UI, etc.